content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -151,7 +153,14 @@ stages:
 by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
 the possible following fields:
 
-- `cmd` (always present): Executable command defined in this stage
+- `cmd` (always present): Executable commands defined in this stage. Each
+  command is passed as is to the shell. The command can be either a single
+  command or a list of commands. In that latter case:
```

Comment:
> Each command is passed as is to the shell.

This suggestion removed that part because it seems obvious unless I'm not understanding a special meaning of this.

```suggestion
- `cmd` (always present): One or more commands executed by the stage
  (may contain either a single value, or a list).
```
> May need formatting

I'm not sure about the "may contain either a single value, or a list" part though, see Q in https://github.com/iterative/dvc.org/pull/1980#pullrequestreview-544313656 above.